jupyterhub/templates/hub/deployment.yaml

```diff
@@ -67,6 +70,37 @@ spec:
         {{- if .Values.hub.extraContainers }}
         {{- .Values.hub.extraContainers | toYaml | trimSuffix "\n" | nindent 8 }}
         {{- end }}
+        - name: consul
```

Comment:
What if we already run consul? Seems redundant to have this configured, especially since it doesn't have TLS, etc.